src/config/ci-backfill.toml

```diff
@@ -11,5 +11,5 @@ streaming_upload_attempt_timeout_ms = 20000
 # NOTE(kwannoel): If can't reproduce it, set to a lower number.
 # This will throttle snapshot read.
 # barrier_interval_ms = 1
-barrier_interval_ms = 50
```

Comment:
Ditto. I think we should change the validation logic for `barrier_interval_ms`. Increasing the configuration here will lead to a longer ci test time.